src/networks.json

```diff
@@ -18,6 +18,26 @@
       "symbol": "WETH"
     }
   },
+  
+"42161": {
```

Comment:
@davidmansion thanks for a pull request. Could you please format this code so it follows the rest of the file.